{{ cookiecutter.name }}/Makefile

```
@@ -0,0 +1,25 @@
+NUMPROCESSES = 4
```

```suggestion
NUMPROCESSES = auto
```

Так pytest-xdist будет всегда использовать максимальное количество CPU